packages/fx-core/src/plugins/resource/frontend/env.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import * as dotenv from "dotenv";
+import fs from "fs-extra";
+import * as path from "path";
+import * as os from "os";
+import { TelemetryHelper } from "./utils/telemetry-helper";
+import { TelemetryEvent } from "./constants";
+
+export interface RemoteEnvs {
+  teamsfxRemoteEnvs: { [key: string]: string };
+  customizedRemoteEnvs: { [name: string]: string };
+}
+
+export const envFileName = (envName: string): string => `env.teamsfx.${envName}`;
+export const envFilePath = (envName: string, folder: string): string =>
+  path.join(folder, envFileName(envName));
+
+export const EnvKeys = Object.freeze({
+  FuncEndpoint: "REACT_APP_FUNC_ENDPOINT",
+  FuncName: "REACT_APP_FUNC_NAME",
+  RuntimeEndpoint: "REACT_APP_TEAMSFX_ENDPOINT",
+  StartLoginPage: "REACT_APP_START_LOGIN_PAGE_URL",
+  ClientID: "REACT_APP_CLIENT_ID",
+});
+
+export function initEnvs(): RemoteEnvs {
```

Comment:
Nit: naming refine, it looks to me that the initEnvs doesn't do any action, so it would be better not to use the verb `init`.
Since it just return a default config object, how about naming it, for example, defaultEnvsConfig, initialEnvsConfig, even emptyEnvsConfig?

Reply:
Good catch! Will update.